src/overview/pricing.md

```diff
@@ -108,3 +109,7 @@ Please [contact our sales department](https://platform.sh/contact/#form) to disc
 The Platform.sh Agency Plan (PAP) waives developer license fees, lets anyone on your team start projects, and permits all-hands access to any site. In addition you get a 10% commission when you recommend Platform.sh to your customers.
 
 Please [contact our sales department](https://platform.sh/contact/#form) to discuss how we can help you.
+
+## Sovereign German Cloud and Australia
+
+The prices shown in the estimation page are for our default cloud regions (EU and US). The Sovereign German Cloud region and the Australian region have specific pricing rules.
```

Comment:
Suggest: "different pricing"

also, what does "the estimation page" mean? https://platform.sh/pricing ?